packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -14,6 +14,55 @@ import type { ThemeOptions } from "@mui/material";
 //import radioClasses from "@mui/material";
 
 export const components: ThemeOptions["components"] = {
+  MuiAlert: {
+    styleOverrides: {
+      root: ({ ownerState, theme }) => ({
+        padding: theme.spacing(4),
+        color: theme.palette.text.primary,
+        ...(ownerState.severity && {
```

Comment:
Rather than describe the individual styles using `variants`, I'm using `ownerState` here to generate the appropriate colors for every available severity.